tutorials/quick_start/pytorch_fw/ultralytics/README.md

```diff
@@ -0,0 +1,42 @@
+# MCT Quick Start - Ultralytics Example 
+
+This example demonstrates how to use MCT 'quick start' with Ultralytics object detection model.
+
+
+## Getting Started
+
+### Installation 
+Install the required library using `pip`:
+```bash
+pip install ultralytics==8.0.47
+ ```
+
+### Usage Examples
+In the following example, we are evaluating the MCT on pre-trained yolov8n, taken from Ultralytics models library
+```python
+python main.py --model_name yolov8n --model_library ultralytics --representative_dataset_folder <my path> --num_representative_images 64 --batch_size 1 
+```
+In this example, we are running the main.py script with the following parameters:
```

Comment:
Add a full commandline example, including the default representative dataset folder to use (../datasets/coco)